log/clog.go

```diff
@@ -0,0 +1,15 @@
+package log
+
+import (
+	"github.com/armory-io/go-commons/server"
+	"github.com/armory-io/go-commons/temporal"
+	"go.uber.org/zap"
+)
+
+func NewClog(ctx temporal.LoggingValuer) *zap.SugaredLogger {
```

Comment:
I would not import an interface like this - it probably feels weird but I would redeclare the interface here. In Go you should (generally) define the interface at the place where you're going to consume the interface, even if you have to redeclare the interface in multiple places. https://github.com/golang/go/wiki/CodeReviewComments#interfaces

Reply:
Thanks I already redeclare the interface here, thanks for the Feedback